daemon/top_windows.go

```diff
@@ -0,0 +1,11 @@
+package daemon
```

Comment:
The list of supported file postfixes is the same as GOOS: http://golang.org/doc/install/source#environment